### PR TITLE
NetworkController PowerShell reference update

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
[1871160](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1871160)

20 new cmdlets and 135 updates to others (plus updates to the module topic.)

### CODE REVIEW

**Comment by TimShererWithAquent** [comment]
@tiburd This one is ready for review.

**Comment by TimShererWithAquent** [comment]
Restarting build.

File: docset/winserver2022-ps/networkcontroller/Get-NetworkControllerFabricRoute.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ title: Get-NetworkControllerFabricRoute
 # Get-NetworkControllerFabricRoute
 
 ## SYNOPSIS
-Gets a network route or all the routes in a subnet in the Network Controller.
+Gets gets the routes specified for logical subnet children of the logical network objects.
```
Comment (get-itips) [inline docset/winserver2022-ps/networkcontroller/Get-NetworkControllerFabricRoute.md:14]:
```suggestion
Gets the routes specified for logical subnet children of the logical network objects.
```

File: docset/winserver2022-ps/networkcontroller/Get-NetworkControllerLoadBalancerBackendAddressPool.md [hunk 2]
```diff
@@ -127,9 +127,13 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
 ## INPUTS
 
+### System.String[]
+
 ## OUTPUTS
 
-### 
+### System.Object
+
+### System.Object#https://localhost/
```
Comment (get-itips) [inline docset/winserver2022-ps/networkcontroller/Get-NetworkControllerLoadBalancerBackendAddressPool.md:136]: can you check this line?

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerRestore.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ConnectionUri
+Specifies the Uniform Resource Identifier (URI) of a Network Controller.
+The cmdlet gets the database restore for that controller.
```
Comment (get-itips) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerRestore.md:57]: Probably this description belongs to the `Get-` cmdlet

File: docset/winserver2022-ps/networkcontroller/Get-NetworkControllerLoadBalancerBackendAddressPool.md (hunk 2 above)
Comment (TimShererWithAquent) [reply docset/winserver2022-ps/networkcontroller/Get-NetworkControllerLoadBalancerBackendAddressPool.md:136]: I removed this line in the new cmdlets per product team input. I'll remove them from these revised ones in bulk.

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerRestore.md (hunk 3 above)
Comment (TimShererWithAquent) [reply docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerRestore.md:57]: Fixing to say removes.

**Review by get-itips — APPROVED** [review]

File: docset/winserver2022-ps/networkcontroller/New-NetworkControllerLoadBalancerConfiguration.md [hunk 4]
```diff
@@ -188,16 +207,50 @@ Type: psobject
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/New-NetworkControllerLoadBalancerConfiguration.md:248]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/New-NetworkControllerLoadBalancerInboundNatRule.md [hunk 5]
```diff
@@ -208,9 +227,45 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/New-NetworkControllerLoadBalancerInboundNatRule.md:261]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/New-NetworkControllerLoadBalancerMux.md [hunk 6]
```diff
@@ -201,9 +220,45 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/New-NetworkControllerLoadBalancerMux.md:254]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/New-NetworkControllerNetworkInterface.md [hunk 7]
```diff
@@ -239,9 +258,45 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/New-NetworkControllerNetworkInterface.md:292]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/New-NetworkControllerPublicIpAddress.md [hunk 8]
```diff
@@ -179,9 +198,45 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/New-NetworkControllerPublicIpAddress.md:232]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/New-NetworkControllerRoute.md [hunk 9]
```diff
@@ -182,13 +201,49 @@ Aliases:
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/New-NetworkControllerRoute.md:239]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/New-NetworkControllerVirtualServer.md [hunk 10]
```diff
@@ -213,9 +232,47 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/New-NetworkControllerVirtualServer.md:266]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerIpPool.md [hunk 11]
```diff
@@ -161,14 +180,52 @@ Aliases:
...
+
+Required: False
+Position: Named
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerIpPool.md:218]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerLoadBalancerFrontendIpConfiguration.md [hunk 12]
```diff
@@ -145,14 +164,52 @@ Aliases:
...
+
+Required: False
+Position: Named
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerLoadBalancerFrontendIpConfiguration.md:202]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerLoadBalancerOutboundNatRule.md [hunk 13]
```diff
@@ -145,14 +164,52 @@ Aliases:
...
+
+Required: False
+Position: Named
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerLoadBalancerOutboundNatRule.md:202]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerLoadBalancerProbe.md [hunk 14]
```diff
@@ -145,14 +164,52 @@ Aliases:
...
+
+Required: False
+Position: Named
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerLoadBalancerProbe.md:202]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerLoadBalancingRule.md [hunk 15]
```diff
@@ -145,14 +164,52 @@ Aliases:
...
+
+Required: False
+Position: Named
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerLoadBalancingRule.md:202]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerMacPool.md [hunk 16]
```diff
@@ -130,14 +149,37 @@ Aliases:
...
+
+Required: False
+Position: Named
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerMacPool.md:172]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerNetworkInterface.md [hunk 17]
```diff
@@ -130,14 +149,52 @@ Aliases:
...
+
+Required: False
+Position: Named
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerNetworkInterface.md:187]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerPublicIpAddress.md [hunk 18]
```diff
@@ -130,14 +149,52 @@ Aliases:
...
+
+Required: False
+Position: Named
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerPublicIpAddress.md:187]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerRoute.md [hunk 19]
```diff
@@ -153,10 +172,48 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerRoute.md:206]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerRouteTable.md [hunk 20]
```diff
@@ -138,10 +157,47 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerRouteTable.md:191]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerServiceInsertion.md [hunk 21]
```diff
@@ -134,10 +152,48 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerServiceInsertion.md:186]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerVirtualNetwork.md [hunk 22]
```diff
@@ -130,14 +149,52 @@ Aliases:
...
+
+Required: False
+Position: Named
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerVirtualNetwork.md:187]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerVirtualSubnet.md [hunk 23]
```diff
@@ -143,14 +162,51 @@ Aliases:
...
+
+Required: False
+Position: Named
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Remove-NetworkControllerVirtualSubnet.md:200]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/networkcontroller/Set-NetworkControllerVirtualSwitchConfiguration.md [hunk 24]
```diff
@@ -199,9 +218,43 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/networkcontroller/Set-NetworkControllerVirtualSwitchConfiguration.md:252]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by dariomws — APPROVED** [review]
Minor suggestions.

**Comment by TimShererWithAquent** [comment]
@tseward Hi - Have you had a chance to look at this one? thanks -

**Review by ghost — APPROVED** [review]

**Comment by TimShererWithAquent** [comment]
Hi @tiburd - This one is approved and ready to merge.

File: docset/winserver2022-ps/networkcontroller/Enable-NetworkControllerNode.md [hunk 25]
```diff
@@ -105,6 +105,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -IntentRejoin
+Indicates that the intent to rejoin.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/networkcontroller/Enable-NetworkControllerNode.md:109]:
```suggestion
Indicates the intent to rejoin.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @TimShererWithAquent, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by TimShererWithAquent** [comment]
Thanks @JohanFreelancer9 - changes accepted.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**